gno.land/cmd/gnoland/start.go

```diff
@@ -372,3 +381,23 @@ func getTxEventStoreConfig(c *startCfg) (*eventstorecfg.Config, error) {
 
 	return cfg, nil
 }
+
+func initTelemetry(ctx context.Context) error {
+	var options []telemetry.Option
+
+	if os.Getenv("TELEM_METRICS_ENABLED") == "true" {
+		options = append(options, telemetry.WithOptionMetricsEnabled())
+	}
+
+	if os.Getenv("TELEM_USE_FAKE_METRICS") == "true" {
```

Comment:
What is the rational to send there fake metrics? is it just in the initial phase for testing?

Reply:
Exactly. For example, when running locally there are no transactions so block building times are always about 0ms so it is hard to tell whether or not the metrics are being collected.

Reply:
I understand the reasons but the metric collection is the response of an event that happen within a system under observation. In other words, if there is no event to observe there should not be any collection. It is counter intuitive sending out "fake" metrics in response of an event that did not happen.  

> when running locally there are no transactions so block building times are always about 0ms so it is hard to tell whether or not the metrics are being collected.

if you want to signal the fact that the system has been bootstrap and hence the building block times is `0ms` then you need to bootstrap that metric and send it out with `0ms` value. This is already a response that communicate clearly to the external worlds that the system has been bootstraped.

Reply:
Removed